docs/source/installation.mdx

```diff
@@ -38,8 +40,8 @@ Currently for Windows users, you need to build bitsandbytes from source:
 
 ```bash
 git clone https://github.com/TimDettmers/bitsandbytes.git && cd bitsandbytes/
-cmake -B build -DBUILD_CUDA=ON -S .
-cmake --build build --config Release
+cmake -DCOMPUTE_BACKEND=cuda
```

Comment:
```suggestion
cmake -DCOMPUTE_BACKEND=cuda -S .
```
For some reason I had to do this on my machine

Reply:
It doesn't make any difference on my machine, but since it doesn't hurt I added it back